overlord/snapstate/handlers.go

```diff
@@ -515,11 +515,29 @@ func (m *SnapManager) doMountSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	cleanup := func() {
+		st.Lock()
+		defer st.Unlock()
+
+		otherInstances, err := hasOtherInstances(st, snapsup.InstanceName())
```

Comment:
nitpick, `otherInstances` feels like there is a collection of other snaps here. Can we call this `ok` or something similar?